.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -1271,3 +1271,25 @@ stages:
       TAGS_TO_SKIP: "os=windows"
     jobs:
       - template: ./templates/e2e-template.yaml
+  - stage: cveQuery
+    condition: ne(variables.SKIP_CVE_QUERY, 'true')
+    jobs:
+      - job: cveQuery
+        pool: $(QUERY_POOL_NAME)
```

Comment:
if this is coming from a variable group can you explicitly declare the group at the top of the yaml to minimize confusion?